src/System Application/App/Password/src/PasswordDialog.Page.al

```diff
@@ -27,6 +27,10 @@ page 9810 "Password Dialog"
                 ExtendedDatatype = Masked;
                 ToolTip = 'Specifies the current password, before the user defines a new one.';
                 Visible = ShowOldPassword;
+                trigger OnValidate()
+                begin
+                    PasswordDialogImpl.ValidateOldPasswordMatch(OldPasswordToCompare, OldPasswordValue);
```

Comment:
Shouldn't we also do the same check `if RequiresPasswordValidation then...` here?

Reply:
As far as I understood from the discussion of the problem it was decided that this rule should always be active. Besides, `RequiresPasswordValidation` is now used only to check the strength of the password and the user has the option not to enable it. 

Based on all this information, it seems to me that I don't need to be bound to `RequiresPasswordValidation`